website/docs/reference/glossary.md

```diff
@@ -0,0 +1,140 @@
+---
+sidebar_position: 2
+---
+
+# Glossary
+
+## List of FerretDB terminologies
+
+*This section contains a list of common terminologies related to FerretDB*.
+
+### A
+
+#### aggregation
+
+A group of operations used to reduce or merge large data sets.
+See [list of supported aggregation operations and commands here](./supported_commands.md#aggregation-pipelines).
+
+#### aggregation pipeline
+
+A set of operators that lets you perform complex operations that aggregate and summarize values.
+See [list of supported aggregation pipeline operators](./supported_commands.md#aggregation-pipeline-operators) here.
+
+---
+
+### B
+
+#### Beacon
+
+The telemetry service of FerretDB.
+See [telemetry](../telemetry.md) for more details.
+
+#### BSON
+
+BSON is a serialized binary file format for storing JSON-like documents.
+
+#### BSON types
+
+The list of types that the BSON format supports.
+BSON offers support for additional data types compared to JSON, such as `timestamp`, `date`, `ObjectId`, and `binary`.
+
+---
+
+### C
+
+#### collection
+
+A group of documents in a non-relational database.
+It is comparable to a table in a relational database.
+
+#### CRUD
+
+The four basic operations of a database: Create, Read, Update, and Delete.
+See [Basic FerretDB CRUD operations here](../basic_operations/index.md).
+
+---
+
+### D
+
+#### database
+
+An organized repository for collections containing its own sets of documents, and data.
+
+#### database command
```

Comment:
Hmm, do we separate "database commands" and "collection commands"? Or do we just call everything a "database command"? Or just a "command"?

Reply:
I believe database commands cover for everything in this regard.